ui/pages/onboarding-flow/onboarding-flow.js

```diff
@@ -199,6 +194,11 @@ export default function OnboardingFlow() {
         );
       }
 
+      // For social login, we need to agree to the terms of use
+      if (isSocialLogin) {
+        await dispatch(setTermsOfUseLastAgreed(new Date().getTime()));
```

Comment:
This isn't necessary because we already set this flag in welcome-login.tsx irrespective of social or srp login